src/flows/Upload/Upload.tsx

```diff
@@ -1,24 +1,41 @@
 import { Box } from "ink"
 import React, { useState } from "react"
 
+import { GameType } from "../../generated/types.js"
+
 import { ChooseGameStep } from "./ChooseGameStep.js"
 import { CreateGameStep } from "./CreateGameStep.js"
 import { CreateGameVersionStep } from "./CreateGameVersionStep.js"
 import { GameDirInputStep } from "./GameDirInputStep.js"
 
 export function Upload() {
   const [gameDir, setGameDir] = useState<string | undefined>()
+  const [multiplayer, setMultiplayer] = useState<boolean>(false)
   const [gameId, setGameId] = useState<number | null | undefined>()
 
   return (
     <Box flexDirection="column">
-      <GameDirInputStep onComplete={setGameDir} />
+      <GameDirInputStep
```

Comment:
Could you explain how these steps work? As in how the cli knows that this is a step and only one of them has to be rendered at once etc.

Unrelated:

Can a singleplayer game become multiplayer?